types/config.go

```diff
@@ -31,6 +32,7 @@ var (
 		coinType:           CoinType,
 		fullFundraiserPath: FullFundraiserPath,
 		txEncoder:          nil,
+		keyringServiceName: "gaia",
```

Comment:
is this supposed to be fixed?

Reply:
where should this be defined?

Reply:
Let's define it as a constant at the top of the file (between the `import` and `Config`):

```go
const DefaultKeyringServiceName = "cosmos"
```

And then move the `sdkConfig` variable into `GetConfig()` as it's not used anywhere else:

```go
// GetConfig returns the config instance for the SDK.
func GetConfig() *Config {
	return &Config{
		sealed: false,
		bech32AddressPrefix: map[string]string{
			"account_addr":   Bech32PrefixAccAddr,
			"validator_addr": Bech32PrefixValAddr,
			"consensus_addr": Bech32PrefixConsAddr,
			"account_pub":    Bech32PrefixAccPub,
			"validator_pub":  Bech32PrefixValPub,
			"consensus_pub":  Bech32PrefixConsPub,
		},
		coinType:           CoinType,
		fullFundraiserPath: FullFundraiserPath,
		txEncoder:          nil,
		keyringServiceName: DefaultKeyringServiceName,
	}
}
```